.travis.yml

```diff
@@ -7,8 +7,12 @@ php:
   - nightly
   - hhvm
 
-install:
+script: phpunit
+
+before_script:
```

Comment:
can you move this section before "script" so that it goes in a chronological order: "before_script" - "script" - "after_script"